docs/using-with-next.md

```diff
@@ -0,0 +1,135 @@
+
+```json
+{
+  "compilerOptions": {
+    // ...
+    "verbatimModuleSyntax": false
+  }
+}
+```
+
+## Install Shadcn Admin Kit Components
+
+Once our project has been initialized, we can pull the `shadcn-admin-kit` components using the `shadcn` command:
+
+```bash
+npx shadcn@latest add https://marmelab.com/shadcn-admin-kit/r/shadcn-admin-kit-base.json
+```
+
+This will create install the `Shadcn` components inside the `components/ui` directory, the `Shadcn Admin Kit` insite the `components/admin` directory, and utilities inside the `hooks` and `lib` directories.
+
+| Admin components                                               | Utilities                                                     |
+| -------------------------------------------------------------- | ------------------------------------------------------------- |
+| ![](./images/shadcn-admin-kit-next/source-code-components.png) | ![](./images/shadcn-admin-kit-next/source-code-utilities.png) |
+
+## Configure the Admin App
+
+The `components/admin/admin.tsx` component supports the same data providers as the traditional [React Admin version](https://marmelab.com/react-admin/DataProviders.html). The list of [supported backends](https://marmelab.com/react-admin/DataProviderList.html) is available on the React Admin documentation. In this example, we will use the `ra-data-json-server` with fake data for the admin.
+
+First, we will create the `components/AdminApp.tsx` file that will contain our admin app. we use the installed `ListGuesser` and `EditGuessers` from `Shadcn Admin Kit` in our two resources.
+
+```tsx
+// components/AdminApp.tsx
+
+"use client"; // remove this line if you choose Pages Router
+import { Resource } from "ra-core";
+import jsonServerProvider from "ra-data-json-server";
+import { Admin, EditGuesser, ListGuesser } from "./admin";
+
+const dataProvider = jsonServerProvider("https://jsonplaceholder.typicode.com");
+
+const AdminApp = () => (
+  <Admin dataProvider={dataProvider}>
+    <Resource
+      name="users"
+      list={ListGuesser}
+      edit={EditGuesser}
+      recordRepresentation="name"
+    />
+    <Resource
+      name="posts"
+      list={ListGuesser}
+      edit={EditGuesser}
+      recordRepresentation="title"
+    />
+    <Resource name="comments" list={ListGuesser} edit={EditGuesser} />
+  </Admin>
+);
+
+export default AdminApp;
+```
+
+Then we can create the `components/Admin.tsx` by using the same code as the [React Admin Next.js](https://marmelab.com/react-admin/NextJs.html) integration tutorial.
+
+```tsx
+// components/Admin.tsx
+
+"use client";
+import dynamic from "next/dynamic";
+
+const Admin = dynamic(() => import("./AdminApp"), {
+  ssr: false, // Required to avoid react-router related errors
+});
+
+export default Admin;
+```
+
+## Create the Admin Page
+
+Once the components have been configured, you can create the admin page at `app/admin/page.tsx`:
+
+```tsx
+// app/admin/page.tsx
+
+import Admin from "@/components/Admin";
```

Comment:
This is very confusing, compared to imports from `"@/components/admin"`. You should find another name.